Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[workspace]
+members = ["components/rust", "profiling"]
+
+[profile.dev]
+debug = 2 # full debug info
+
+[profile.release]
+debug = 1
+lto = "thin"
+incremental = false
+
+[profile.tracer-release]
+debug = 1 # line tables only
+lto = true
+codegen-units = 1
+opt-level = "z" # optimize for size
```

Comment:
How many bytes are you saving here? I don't think the profiler would want to be optimized for size instead of speed (although tests could determine the extent of this).

Reply:
"z" = 6799000
"s" = 6279360

Reply:
I'm going to omit it...